<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>深度学习训练模型</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="btn_train">
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>训练模型</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_import"/>
    <addaction name="action_delete"/>
   </widget>
   <widget class="QMenu" name="btn_detect">
    <property name="title">
     <string>检测数据</string>
    </property>
    <addaction name="action1_new"/>
    <addaction name="action2_import"/>
    <addaction name="action_delete_2"/>
   </widget>
   <widget class="QMenu" name="btn_labeling">
    <property name="title">
     <string>数据标注</string>
    </property>
    <addaction name="actionsss"/>
   </widget>
   <widget class="QMenu" name="menuff">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action13344"/>
   </widget>
   <addaction name="btn_train"/>
   <addaction name="btn_detect"/>
   <addaction name="btn_labeling"/>
   <addaction name="menuff"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_new">
   <property name="text">
    <string>新建训练工程</string>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_import">
   <property name="text">
    <string>导入已有工程</string>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除训练工程</string>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action1_new">
   <property name="text">
    <string>新建测试工程</string>
   </property>
  </action>
  <action name="action2_import">
   <property name="text">
    <string>导入测试工程</string>
   </property>
  </action>
  <action name="action_delete_2">
   <property name="text">
    <string>删除测试工程</string>
   </property>
  </action>
  <action name="action4_conbime">
   <property name="text">
    <string>联合测试</string>
   </property>
  </action>
  <action name="actionsss">
   <property name="text">
    <string>开始标注</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action13344">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
